www/include/configuration/configObject/service_dependency/getServiceXml.php

```diff
@@ -107,7 +107,7 @@
 					SELECT s.service_id, s.service_description, h.host_name, h.host_id 
 					FROM service s, host h, host_service_relation hsr 
 					WHERE hsr.hostgroup_hg_id IS NULL 
-                                        AND h.host_id = '" . $db->escape($hostId). "' 
+                                        AND h.host_id = " . (int)$hostId . " 
```

Comment:
you could avoid sending a useless query by checking if hostId is false on line 87 -> 
https://github.com/centreon/centreon/pull/8758/files#diff-2301f8bfcef29a6676ab357c678d03d8R87